pySDC/implementations/problem_classes/AdvectionEquation_ND_FD.py

```diff
@@ -187,24 +204,27 @@ def u_exact(self, t, **kwargs):
             t (float): current time
 
         Returns:
-            dtype_u: exact solution
+            me: exact solution
```

Comment:
What's your scheme for this? The name of the variable is not returned, so I personally care only about the order and what it is when looking at the return values. I get that when looking though a function this can help. What about `me (dtype_u): exact solution`?